README.md

```diff
@@ -0,0 +1,27 @@
+# CAOTIC
+High Contrast and Adaptive Optics laboratory testbeds community website
+
+Welcome to the community website for astronomical testbeds! We are currently working on getting our 
+website migrated to this place here, the original website can be found here:  
+https://sites.google.com/view/highcontrastlabs/
+
+## Local setup - MacOS
+
+### Initial setup
+
+1. install homebrew  
+https://osxdaily.com/2018/03/07/how-install-homebrew-mac-os/
+
+2. install ruby  
+`$ brew install ruby`
+
+3. install bundler  
+`$ gem install bundler`
```

Comment:
Hit a problem here as well with permissions. I assume this is a problem just on my end so I can reach out to IT, but I just want to confirm that this the correct command

Reply:
Yes it is, and I am a little surprised there's a permissions error as this is just a ruby gem installation, *after* it had to allow you to install ruby, so this is definitely also good to know.

Reply:
Update: Need to include a step between 2 and 3 telling the user to run `echo 'export PATH="/usr/local/opt/ruby/bin:$PATH"' >> /Users/<your-username-here>/.bash_profile`. I don't know if you need to source it, but I did and things worked when I did that